### ISSUE TITLE
ProductSearch sample function fails to run

### ISSUE THREAD

**Issue by moody31415** [OfficeDev/script-lab#1011]
### Bug Report

- **Host**: EXCEL
- **OS**: Windows 11 23H2; 22631.2048
- **Browser**: Edge production
- **Environment**: Production

#### Expected behavior:

Script lab doesn't give errors

#### Actual behavior:

Formula results in a #BUSY! error that never resolves; flashes a dozen+ times, and scriptlab shows "add-in error" business bar

#### Steps to Reproduce:

1. Open Excel; install scriptlab addin; new blank workbook
2. Search samples for "data types: custom functions"; open that sample
3. Register that sample
4. As described in the samples' code / instructions, paste this into an arbitrary cell in your spreadsheet:
5. =SCRIPTLAB.DATATYPESCUSTOMFUNCTIONS.PRODUCTSEARCH("chef", false)

Expect: Script to run, show results
Actual: See persistent #BUSY! error that flickers a number of times, and "Add-in error" message shows up within scriptlab. Clicking "restart" in the business bar causes it to dismiss very briefly, then show up again.

#### Failure Logs:
No failure messages are shown in scriptlab's console.

#### Screenshot:
![image](https://github.com/OfficeDev/script-lab/assets/15916159/82970980-6702-4dc4-9e5f-287e71f6e856)
![image](https://github.com/OfficeDev/script-lab/assets/15916159/7f1c5874-0451-443f-aed6-80306bde989a)

**Comment by moody31415** [comment]
This also seems to leave Excel in a "trying to figure something out" state, as I can see through task manager that it's chewing up a lot of CPU after inserting that method that is failing to resolve.

**Comment by rulingthewaves** [comment]
Same problem for me - different script different function, same error.

The code loads and is perfectly fine (unchanged from the code I was using the last few weeks), but the add-in just doesn't start. Have tried even basic scripts from the gitlabs library, problem definitely with Excel or the Script Lab add-in. Seems one of them changed / background updated in the last 24 hours.

**Comment by Gapplay** [comment]
Yeah, I've tried office 2016 and 365. It gives the same error "Add-in Error: This add-in could not be started."

**Comment by GerousDan** [comment]
On Excel 365 Windows 9on various machines) all samples for "data types: custom functions" (which used to work before) now give 'ADD-IN ERROR This add-in could not be started. Close this dialog to ignore the problem or click "Restart" to try again.' No suggestions including updating, repairing, reinstalling Office, not even after running as admin 'powershell Add-AppxPackage -register -disable c:\windows\SystemApps\Microsoft.Win32WebViewHost_cw5n1h2txyewy\AppxManifest.xml', not even repairing/reinstalling WebView2 helped at all and custom function samples from the Script Lab add-in all fail with the same error.

Excel 365 on MacOSX runs Script Lab sample custom functions perfectly as before.

**Comment by chrisloveit** [comment]
Hi,

This is similar to the following issue: https://github.com/OfficeDev/script-lab/issues/1014

I am encountering the exact same issue with Excel and Script Lab 😔

Here are my details:
- **Operating system**: Windows 10 Professional, build 19045.3208
- **Excel**: Microsoft® Excel® for Microsoft 365 MSO (Version 2307 Build 16.0.16626.20170) 64-bit

**Comment by novikov-alexander** [comment]
The same problem exists for me. 
The add-in works for regular code snippets, but even the simplest custom functions fail.

Excel: version 2310 (build 16924.20106)
